addon/services/flash-messages.js

```diff
@@ -140,7 +142,8 @@ export default Service.extend({
   },
 
   _enqueue(flashInstance) {
-    const preventDuplicates = get(this, 'defaultPreventDuplicates');
+    const preventDuplicates = get(this, 'defaultPreventDuplicates') ||
+      get(flashInstance, 'preventDuplicates');
```

Comment:
I'd have imagined that the option on the `flashInstance` itself should trump the global setting as it's more specific. This would not be the case with the above logic. What's the intention?

Reply:
You're right, that makes more sense.
I'll change it soon